src/atoms/LiistSection.js

```diff
@@ -39,13 +42,11 @@ export class LiistSection extends LitElement {
 
   render() {
     return html`
-    <div class="container">
-      <p><slot name="title" id="title">loading ... </slot></p>
-      <div class="line"></div>
-      <div class="content">
-        <slot name="fill-content"></slot>
+      <div class="container">
+        <p id="title">${this.title}</p>
+        <div class="line"></div>
+        <slot></slot>
```

Comment:
A very easy pattern is to just place the slot with no attributes at all and then everything that is put in between the opening and closing html tag of the component gets put there. No need to connect html elements with matching name properties. 

Also i put the title as property, not a slot, because of the same reason than in the `LiistBttn`.